app/assets/javascripts/admin/components/admin-editable-field.js.es6

```diff
@@ -0,0 +1,23 @@
+export default Ember.Component.extend({
+  tagName: "",
+
+  buffer: "",
+  editing: false,
+
+  init() {
+    this._super();
```

Comment:
it's not crucial, but if you think about it

`this._super(...arguments)` is better